integration/test_auditlogs.py

```diff
@@ -3,13 +3,38 @@
 
 import pytest
 from integration import run_command
+from integration.util import DockerDaemon
+from integration.util import SyslogServer
+
 
 BASE_COMMAND = "code42 audit-logs search -b"
 begin_date = datetime.utcnow() - timedelta(days=-10)
 begin_date_str = begin_date.strftime("%Y-%m-%d %H:%M:%S")
 
 
+@pytest.fixture
+def data_transfer():
+    with DockerDaemon():
+        with SyslogServer():
+            yield run_command
+
+
 @pytest.mark.parametrize("command", [("{} '{}'".format(BASE_COMMAND, begin_date_str))])
 def test_auditlogs_search(command):
     return_code, response = run_command(command)
     assert return_code == 0
+
+
+@pytest.mark.parametrize(
```

Comment:
Why is this parameterized? 
in the case where we want to add more commands, we don't to include `code42 audit-logs sent-to` on every parameter since it does not change

Reply:
At that time I had thought of 2 commands to be tested one with `TCP` protocol option and another with `UDP`. Now that we have `TCP-TLS` having parameterize makes more sense. 

 Moreover, using parameterize even for a single test case is useful while debugging errors. With parameterize option we can see the actual command executed in this particular case... Below is a sample output of `pytest -v` 
 
 `tests/integration/test_auditlogs.py::test_auditlogs_send_to[code42 audit-logs send-to localhost -p TCP -b '2021-01-19 13:32:46'] ERROR                                                               [  4%]
tests/integration/test_auditlogs.py::test_auditlogs_search_command_returns_success_return_code[code42 audit-logs search -b '2021-01-19 13:32:46'] FAILED                                             [  9%]
tests/integration/test_auditlogs.py::test_auditlogs_search_command_returns_success_return_code[code42 audit-logs search -b '2021-01-19 13:32:46' -e '2021-01-21 13:32:46']`